src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.hh

```diff
@@ -117,6 +118,7 @@ struct CoilCoolingDXCurveFitPerformance
     std::string name;
     Real64 crankcaseHeaterCap = 0.0;
     Real64 crankcaseHeaterPower = 0.0;
+    Real64 crankcaseHeaterCapacityCurveIndex = 0.0;
```

Comment:
Integer

Reply:
It was fixed in another commit. I'll push it soon. I was trying to figure out some issues with the variable speed coils curve application.

Reply:
> It was fixed in another commit.

This is still not an integer.  Did you by chance forget to push a commit?